### PR TITLE
Fixes to security questions

### PR DESCRIPTION
**Pull request by JGBJSS**

### CODE REVIEW

File: src/main/resources/features/theme 1/20 - forgotten card.feature [hunk 1]
```diff
@@ -6,18 +6,17 @@ Feature: 20 - Existing user has forgotten 2FA card
     Given I login and click "Lost, forgotten or damaged security" card using "2FA_CARD_USER" as {username}, {lastDrift}, {question1}, {question2}
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/20 - forgotten card.feature:6]: Do we still need the SQL returning the {question1} and {question2} data?

File: src/main/resources/features/theme 1/21 - lost card.feature [hunk 2]
```diff
@@ -6,34 +6,33 @@ Feature: 21 - Existing user has lost 2FA card
     Given I login and click "Lost, forgotten or damaged security" card using "2FA_CARD_USER" as {username}, {lastDrift}, {question1}, {question2}
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/21 - lost card.feature:6]: Do we still need the SQL returning the {question1} and {question2} data?

File: src/main/resources/features/theme 1/21 - lost card.feature [hunk 3]
```diff
@@ -80,40 +78,37 @@ Feature: 21 - Existing user has lost 2FA card
     And I login with 2FA as {username}
     Then The page title contains "Your home"
 
-
-  Scenario: Existing user orders new card via profile screen, then activates it, then logs in
+#  Scenario: Existing user orders new card via profile screen, then activates it, then logs in
     # Login via security questions, change insecure answers to questions
     Given I login and click "Lost, forgotten or damaged security" card using "2FA_CARD_USER" as {username}, {lastDrift}, {question1}, {question2}
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/21 - lost card.feature:83]: Do we still need the SQL returning the {question1} and {question2} data?

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 4]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
     #Q&A1 - Select a question from the dropdown, enter a common answer and submit and verify the correct error is shown.
-    And The page title contains "First security question - MOT testing service"
+    And The page title contains "MOT testing service"
     And I select "What was the name of your first pet?" in the field with id "question1"
     And I enter "answer" in the field with id "answer1"
     And I press the "Continue" button
     Then The page contains "There was a problem with the information you entered:"
     Then The page contains "First memorable answer - must not be a common answer"
+    Then The page contains "Second security question - choose a question from the list"
+    Then The page contains "Second memorable answer - enter a memorable answer"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 5]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
+    Then The page contains "Third security question - choose a question from the list"
+    Then The page contains "Third memorable answer - enter a memorable answer"
     Then The page contains "Must not be a common answer"
+    Then The page contains "Choose a question from the list"
+    Then The page contains "Enter a memorable answer"
+    Then The page contains "Choose a question from the list"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature:29]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 6]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
     Then The page contains "There was a problem with the information you entered:"
     Then The page contains "First memorable answer - must be shorter than 71 characters"
+    Then The page contains "Second security question - choose a question from the list"
+    Then The page contains "Second memorable answer - enter a memorable answer"
+    Then The page contains "Third security question - choose a question from the list"
+    Then The page contains "Third memorable answer - enter a memorable answer"
     Then The page contains "Must be shorter than 71 characters"
+    Then The page contains "Choose a question from the list"
+    Then The page contains "Enter a memorable answer"
+    Then The page contains "Choose a question from the list"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature:43]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 7]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
+    Then The page contains "Enter a memorable answer"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 8]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
     Then The page contains "First memorable answer - must not be a common answer"
+    Then The page contains "Second security question - choose a question from the list"
+    Then The page contains "Second memorable answer - enter a memorable answer"
+    Then The page contains "Third security question - choose a question from the list"
+    Then The page contains "Third memorable answer - enter a memorable answer"
     Then The page contains "Must not be a common answer"
+    Then The page contains "Choose a question from the list"
+    Then The page contains "Enter a memorable answer"
+    Then The page contains "Choose a question from the list"
+    Then The page contains "Enter a memorable answer"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 9]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
     Then The page contains "There was a problem with the information you entered:"
     Then The page contains "First memorable answer - must be at least 6 characters"
+    Then The page contains "Second security question - choose a question from the list"
+    Then The page contains "Second memorable answer - enter a memorable answer"
+    Then The page contains "Third security question - choose a question from the list"
+    Then The page contains "Third memorable answer - enter a memorable answer"
     Then The page contains "Must be at least 6 characters"
+    Then The page contains "Choose a question from the list"
+    Then The page contains "Enter a memorable answer"
+    Then The page contains "Choose a question from the list"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature:57]: Remove the duplication

File: src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature [hunk 10]
```diff
@@ -13,34 +13,58 @@ Feature: 24 - Verify Q&A policy is enabled and working
...
+    Then The page contains "Enter a memorable answer"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/24 - change Q&A according to new policy.feature]: Remove the duplication

**Review by karlbeckettbjss — CHANGES_REQUESTED** [review]

File: src/main/resources/features/theme 1/20 - forgotten card.feature (hunk 1 above)
Comment (karlbeckettbjss) [reply src/main/resources/features/theme 1/20 - forgotten card.feature:6]: No longer Required

File: src/main/resources/features/theme 1/21 - lost card.feature (hunk 2 above)
Comment (karlbeckettbjss) [reply src/main/resources/features/theme 1/21 - lost card.feature:6]: No longer Required

File: src/main/resources/features/theme 1/21 - lost card.feature (hunk 3 above)
Comment (karlbeckettbjss) [reply src/main/resources/features/theme 1/21 - lost card.feature:83]: No longer Required

File: src/main/resources/features/theme 1/20 - forgotten card.feature [hunk 11]
```diff
@@ -6,18 +6,17 @@ Feature: 20 - Existing user has forgotten 2FA card
...
     And The page title contains "Sign in without your security card"
     When I click the "Continue" link
-    And I enter "answer" in the {question1} field
-    And I enter "answer" in the {question2} field
+    And I enter "answer" in the field with id "answer"
     And I press the "Continue" button
     And The page title contains "Change security questions - Reset your account security - MOT testing service"
     And I press the "Continue" button
-    And The page title contains "First security question - MOT testing service"
+    And The page title contains "MOT testing service"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/20 - forgotten card.feature:13]: Should this "changes your security question" - bug in the application

File: src/main/resources/features/theme 1/21 - lost card.feature [hunk 12]
```diff
@@ -6,34 +6,33 @@ Feature: 21 - Existing user has lost 2FA card
...
     And The page title contains "Sign in without your security card"
     When I click the "Continue" link
-    And I enter "answer" in the {question1} field
-    And I enter "answer" in the {question2} field
+    And I enter "answer" in the field with id "answer"
     And I press the "Continue" button
     And The page title contains "Change security questions - Reset your account security - MOT testing service"
     And I press the "Continue" button
-    And The page title contains "First security question - MOT testing service"
+    And The page title contains "MOT testing service"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/21 - lost card.feature:13]: Same above

**Review by ibrahimjassat — APPROVED** [review]

**Review by karlbeckettbjss — APPROVED** [review]

**Pull request merged**